tickets/src/components/content/NewEventContent.tsx

```diff
@@ -53,6 +53,13 @@ export class EventContent extends Component<
     if (typeof window !== 'undefined') {
       window.addEventListener('unlockProtocol', this.setPaywallStatus)
 
+      const url = new URL(
+        window.location.href,
+        'https://tickets.unlock-protocol.com'
```

Comment:
Is this going to break on staging/dev?

Reply:
Based on the usage in `NewDemoContent`, that second piece is just a default base URL to use in case we get a relative url from `window.location.href`. It never makes any network calls, it just uses the string representation to pull out the search param

Reply:
yeah, but since it is going to be a different URL base, this means we cannot parse successful in other env?